.azure-pipelines/Windows-CI.yml

```diff
@@ -52,9 +61,13 @@ jobs:
       git submodule update --init --recursive
       set ONNX_BUILD_TESTS=1
       set ONNX_ML=$(onnx_ml)
-      set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=ON -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=ON
+      if '$(protobuf_type)' == 'External' (
+        set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=ON -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=ON
+      ) else (
+        set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=OFF -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=OFF
+      )
 
-      pip install -e .
+      pip install -e . -v
```

Comment:
Thank you for adding `-v` to make the log more friendly for debugging! Could you please add `-v` for `pip install -e .` in Linux and Mac CI as well?

Reply:
Done